packages/modeling/src/operations/transforms/align.d.ts

```diff
@@ -1,14 +1,14 @@
 import { Geometry } from '../../geometries/types'
-import Vec3 from '../../maths/vec3/type'
+import { Vec1, Vec2, Vec3 } from '../../maths/types'
 import RecursiveArray from '../../utils/recursiveArray'
 
 export default align
 
 export interface AlignOptions {
   modes?: Array<'center' | 'max' | 'min' | 'none'>
-  relativeTo?: Vec3
+  relativeTo?: Vec1 | Vec2 | Vec3
```

Comment:
Actually, this declaration of relativeTo is incorrect. It should just be Array. Using Vec2 or Vec3 implies that those arrays are numeric, which is not correct. This option (if provided) can have any combination of:
- [null] or [number]
- [null, null] or [null, number] or [number, number]
- [null, null, null] or [null, number, null] or [null, null, number] or [null, number, number] or [number, number, number]

Strange indeed...

Reply:
Which brings up my next comment... Vec1 is not necessary now.

Reply:
Thanks. So it's not Vec1, Vec2, or Vec3, but some other type. I'll look at the implementation and try to define the type.